eox_core/settings/aws.py

```diff
@@ -120,7 +120,24 @@ def plugin_settings(settings):  # pylint: disable=function-redefined
         settings.EOX_CORE_SENTRY_INTEGRATION_DSN
     )
     if sentry_sdk is not None and sentry_integration_dsn is not None:
+        def before_send(event, hint):
```

Comment:
For me it makes sense to define this function in a different file.
And also to make the list of ignored errors a configurable value